test/e2e/predictor/test_sklearn.py

```diff
@@ -221,6 +223,32 @@ def test_sklearn_v2():
     res = predict(service_name, "./data/iris_input_v2.json", protocol_version="v2")
     assert res["outputs"][0]["data"] == [1, 1]
 
+    raw_res = predict(
+        service_name,
+        "./data/iris_input_v2_binary.json",
```

Comment:
How do we test sending inputs with binary data for REST ?

Reply:
Added in custom transformer test and also included in python tests